governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -0,0 +1,310 @@
+        )
+        .map(
+          (product) =>
+            (symbolToData[product.metadata.symbol] = {
+              address: product.address.toBase58(),
+              metadata: product.metadata,
+              priceAccounts: product.priceAccounts.map((p) => {
+                return {
+                  address: p.address.toBase58(),
+                  publishers: p.publishers.map((p) => p.toBase58()),
+                  expo: p.expo,
+                  minPub: p.minPub,
+                }
+              }),
+            })
+        )
+      symbolToData = sortData(symbolToData)
+      setData(symbolToData)
+    }
+  }, [rawConfig, dataIsLoading])
+
+  const sortData = (data: any) => {
+    const sortedData: any = {}
+    Object.keys(data)
+      .sort()
+      .forEach((key) => {
+        const sortedInnerData: any = {}
+        Object.keys(data[key])
+          .sort()
+          .forEach((innerKey) => {
+            if (innerKey === 'metadata') {
+              sortedInnerData[innerKey] = sortObjectByKeys(data[key][innerKey])
+            } else if (innerKey === 'priceAccounts') {
+              // sort price accounts by address
+              sortedInnerData[innerKey] = data[key][innerKey].sort(
+                (
+                  priceAccount1: PriceRawConfig,
+                  priceAccount2: PriceRawConfig
+                ) =>
+                  priceAccount1.address
+                    .toBase58()
+                    .localeCompare(priceAccount2.address.toBase58())
+              )
+              // sort price accounts keys
+              sortedInnerData[innerKey] = sortedInnerData[innerKey].map(
+                (priceAccount: any) => {
+                  const sortedPriceAccount: any = {}
+                  Object.keys(priceAccount)
+                    .sort()
+                    .forEach((priceAccountKey) => {
+                      if (priceAccountKey === 'publishers') {
+                        sortedPriceAccount[priceAccountKey] = priceAccount[
+                          priceAccountKey
+                        ].sort((pub1: string, pub2: string) =>
+                          pub1.localeCompare(pub2)
+                        )
+                      } else {
+                        sortedPriceAccount[priceAccountKey] =
+                          priceAccount[priceAccountKey]
+                      }
+                    })
+                  return sortedPriceAccount
+                }
+              )
+            } else {
+              sortedInnerData[innerKey] = data[key][innerKey]
+            }
+          })
+        sortedData[key] = sortedInnerData
+      })
+    return sortedData
+  }
+
+  const sortObjectByKeys = (obj: any) => {
+    const sortedObj: any = {}
+    Object.keys(obj)
+      .sort()
+      .forEach((key) => {
+        sortedObj[key] = obj[key]
+      })
+    return sortedObj
+  }
+
+  // function to download json file
+  const handleDownloadJsonButtonClick = () => {
```

Comment:
Two fields in the downloaded json are not really editable. I suggest you don't show price_account (low-level info).
You can show symbol but you should error if someone tries to mutate.